sigar_openbsd.go

```diff
@@ -282,7 +292,7 @@ func (self *Swap) Get() error {
 
 	rnswap := C.swapctl(C.SWAP_STATS, unsafe.Pointer(&swdev[0]), nswap)
 	if rnswap == 0 {
-		return nil
+		return fmt.Errorf("Failed to call swapctl(2)")
```

Comment:
Will this return an error when a machine has 0 swap devices?  The OpenBSD man page for [swapctl(2)](http://man.openbsd.org/man2/swapctl.2#RETURN_VALUES) is somewhat ambiguous here, but my interpretation is that a return value of 0 is OK and -1 indicates failure.